### ISSUE TITLE
< waiting for any device > But the device is connected

### ISSUE THREAD

**Issue by mori1209** [hackerstech/Google-FRP-Bypass#1]
I working with virtual box and the device is connected, but i get a message < waiting for any device >
I get into Download mode (odin mode)

**Comment by Elyapk** [comment]
yeah mine says the same thing I am only on kali app for windows but it should work the same right?
